VelocidiSDK/VelocidiSDK/VSDKUtil.m

```diff
@@ -3,15 +3,6 @@
 #import "VSDKGlobalVariables.h"
 #import "VSDKUtil.h"
 
-#if defined(__IPHONE_14_0) || defined(__MAC_10_16) || defined(__TVOS_14_0) || defined(__WATCHOS_7_0)
-// Are we in a build tool that has access to the iOS 14 SDK?
-#define CAN_BUILD_FOR_IOS14 1;
-#endif
-
-#if defined(CAN_BUILD_FOR_IOS14)
-@import AppTrackingTransparency; // NOTICE: linking happens automatically when using Modules or "semantic import".
-#endif
-
 @implementation VSDKUtil
 
 NSString * const trackingNotAllowedErrordomain = @"com.velocidi.VSDKTrackingNotAllowedError";
```

Comment:
Can you remove this (and the one below)?

Reply:
Done in f3edbe5